torch_xla_py/xla_model.py

```diff
@@ -109,6 +109,8 @@ def xla_real_devices(devices):
 
 def xla_replication_devices(local_devices):
   real_devices = xla_real_devices(local_devices)
+  if len(real_devices) == 1:
+      return real_devices
```

Comment:
I think the correct behavior is to `return None`, see https://github.com/pytorch/xla/commit/aa9153c4ead2c5bac81a88b0e4eb1a9690ada5b9#diff-de2e1be6d6e1d3b79322ff7b9f2c7d15

Reply:
If I use return None, I get this error:

Traceback (most recent call last):
  File "xla/test/test_train_imagenet.py", line 209, in test_accurracy
    self.assertGreaterEqual(train_imagenet(), FLAGS.target_accuracy)
  File "xla/test/test_train_imagenet.py", line 148, in train_imagenet
    model_parallel = dp.DataParallel(torchvision_model, device_ids=devices)
  File "/usr/share/torch-xla-nightly/pytorch/xla/torch_xla_py/data_parallel.py", line 185, in __init__
    self._replication = xm.Replication(self._device_ids, replication_devices)
  File "/usr/share/torch-xla-nightly/pytorch/xla/torch_xla_py/xla_model.py", line 27, in __init__
    self._replication_devices = list(replication_devices)
TypeError: 'NoneType' object is not iterable

Reply:
That's because you would need to handle that case separately as in the deleted code in https://github.com/pytorch/xla/commit/aa9153c4ead2c5bac81a88b0e4eb1a9690ada5b9#diff-de2e1be6d6e1d3b79322ff7b9f2c7d15

e.g.
```
replication_devices = (
        xm.xla_replication_devices(self._device_ids)
        if self._device_ids else None)
```